rust/connlib/tunnel/src/tests/sut.rs

```diff
@@ -300,10 +300,25 @@ impl TunnelTest {
                 const IDLE_DURATION: Duration = Duration::from_secs(6 * 60); // Ensure idling twice in a row puts us in the 10-15 minute window where TURN data channels are cooling down.
                 let cut_off = state.flux_capacitor.now::<Instant>() + IDLE_DURATION;
 
+                debug_assert_eq!(buffered_transmits.packet_counter(), 0);
+
                 while state.flux_capacitor.now::<Instant>() <= cut_off {
                     state.flux_capacitor.tick(Duration::from_secs(5));
                     state.advance(ref_state, &mut buffered_transmits);
                 }
+
+                let num_packets = buffered_transmits.packet_counter() as f64;
+                let num_connections = state.client.inner().sut.num_connections() as f64 + 1.0; // +1 because we may have 0 connections.
+                let num_seconds = IDLE_DURATION.as_secs() as f64;
+
+                let packets_per_sec = num_packets / num_seconds / num_connections;
+
+                // This has been chosen through experimentation. It primarily serves as a regression tool to ensure our idle-traffic doesn't suddenly spike.
+                const THRESHOLD: f64 = 2.0;
```

Comment:
I am somewhat surprised by this number. I think it should be less than that given that we only send one every 60s when idling.

It is significantly less than before though.

Either my math is wrong or the packets we send to relays are accounting for that many? It is weird.

I'll investigate in a follow-up.